plugin.video.filmarkivet/resources/language/English/strings.xml

```diff
@@ -2,16 +2,13 @@
 <strings>
```

Comment:
Would be cool if you could convert the language xmls to po files at some point.
(no need to it with this PR)